server/http/src/service.rs

```diff
@@ -219,7 +225,25 @@ where
                         .run_query(query)
                         .map_err(|e| GraphQLServerError::from(e))
                 })
-                .then(|result| GraphQLResponse::new(result)),
+                .then(move |result| {
+                    let elapsed = start.elapsed().as_millis();
+                    match result {
+                        Ok(_) => info!(
+                            logger,
+                            "GraphQL query served.";
```

Comment:
Can we remove the `.` at the end of the messages?

Also: the log codes are not being used in these two messages.